lib/index.js

```diff
@@ -31,7 +31,7 @@ var settings = [
   'partials',
   'pattern',
   'rename',
-  'exposeConsolidate'
+  'renderer',
```

Comment:
You don't have to align them. I can do that if I stumble upon the ones where it makes sense.
I'm all in favour of the lint file.